aws_xray_sdk/core/recorder.py

```diff
@@ -18,12 +19,13 @@
 from .daemon_config import DaemonConfig
 from .plugins.utils import get_plugin_modules
 from .lambda_launcher import check_in_lambda
-from .exceptions.exceptions import SegmentNameMissingException
+from .exceptions.exceptions import SegmentNameMissingException, SegmentNotFoundException
 from .utils.compat import string_types
 from .utils import stacktrace
 
 log = logging.getLogger(__name__)
 
+XRAY_ENABLED_KEY = 'AWS_XRAY_ENABLED'
```

Comment:
We discussed changing this to `AWS_XRAY_SDK_ENABLED` in the previous PR comments

Reply:
Forgot to remove. Never actually used. Removed